packages/cherry-pick-bot/src/bot.ts

```diff
@@ -240,6 +246,9 @@ export = (app: Probot) => {
         repo,
         [context.payload.pull_request.merge_commit_sha],
         targetBranch,
+        remoteConfig.preservePullRequestTitle
+          ? `cherry-pick ${context.payload.pull_request.title} (#${context.payload.pull_request.number})`
```

Comment:
This doesn't match the conventional commit.

Reply:
Would adding `cherry-pick` at the end be ok ?

Something like `${context.payload.pull_request.title} (cherry-pick #${context.payload.pull_request.number})`.

Reply:
Yeah looks good

Reply:
Fixed